src/schema/ip_options.rs

```diff
@@ -79,6 +134,8 @@ impl From<StoredFlag> for IpAddrOptions {
 impl From<IndexedFlag> for IpAddrOptions {
     fn from(_: IndexedFlag) -> Self {
         IpAddrOptions {
+            fieldnorms: true,
```

Comment:
maybe we can drop fieldnorms for IP addresses?
what do you think?

Reply:
I would keep it to be able to use BM25